src/Microsoft.Tye.Core/ValidateIngressStep.cs

```diff
@@ -56,14 +56,6 @@ public override async Task ExecuteAsync(OutputContext output, ApplicationBuilder
             output.WriteDebugLine($"Validating ingress class '{ingressClass}'.");
             var config = KubernetesClientConfiguration.BuildDefaultConfig();
 
-            // Workaround for https://github.com/kubernetes-client/csharp/issues/372
-            var store = await KubernetesClientConfiguration.LoadKubeConfigAsync();
```

Comment:
@rynowak I noticed when verifying this locally that config.Namespace is null when using the default namespace. Do you want to still have config.Namespace be "default"?